api/api_autogen/library/defaults/Battery_GenericBatteryMerchantPlant.json

```diff
@@ -37,7 +35,7 @@
 			"batt_replacement_schedule_percent": [0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000],
 			"batt_surface_area": 89.0141,
 			"en_batt": 1,
-			"om_replacement_cost1": [500.000000]
```

Comment:
What happened such that these variables are just now getting updated? I would have expected this change in 2.2.0.

Reply:
Not really sure why. 2.2.0 was built shortly after the release.